bokehjs/src/coffee/models/annotations/color_bar.coffee

```diff
@@ -515,6 +522,7 @@ export class ColorBar extends Annotation
       minor_coords[i] = new Float64Array((scale_length - coord for coord in minor_coords[i]))
 
     return {
+      "major_ticks": major_ticks
       "major": major_coords
       "minor": minor_coords
       "major_labels": major_labels
```

Comment:
Alternatively, slice the other dimension:
```
return {
  major: major
  minor: minor
}
```
Where `major` and `minor` each have `.ticks` `.labels` and `.coords`